localoam/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:4.5
+ENV NPM_CONFIG_LOGLEVEL warn
+
+# Install app dependencies
+ADD package.json /tmp/package.json
+RUN cd /tmp && npm install
+
+# Create app directory
+RUN mkdir -p /usr/src/app && cp -a /tmp/node_modules /usr/src/app
+WORKDIR /usr/src/app
```

Comment:
How about doing this first, copying `package.json` in, and running `npm install` in the current `WORKDIR`?